### PR TITLE
Final/hide follower buttons

### PR DESCRIPTION
**Pull request by shaargtz**
# What does this PR do?
- Reorders logic for showing and hiding things from users whether they're logged in or not.

**Related Issues**
- NA

---
**Merge Checklist**
- [x] Branch is up to date
- [ ] No code style issues

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -555,7 +534,7 @@ function createQuestionElement(question, isForum) {
   // Star icon.
   const iconElement = document.createElement('i');
   iconElement.setAttribute('id', 'icon' + question.id);
-  iconElement.setAttribute('style', 'cursor: pointer');
+  iconElement.setAttribute('style', 'cursor: pointer; display: none');
```
Comment (abundis28) [inline src/main/webapp/script.js:537]: Perhaps a more descriptive name like starIconElement would make it easier to understand. You do have the comment which makes this just a suggestion and not a request.

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -863,6 +842,46 @@ function setQuestionIdValue() {
...
+      if (solidIcon != null) {
+        for (element of solidIcon) {
+          element.style.display = "block";
+        }
+      }
+
+      const outlineIcon = document.getElementsByClassName('far fa-star fa-2x');
+      if (outlineIcon != null) {
+        for (element of outlineIcon) {
+          element.style.display = "block";
```
Comment (abundis28) [inline src/main/webapp/script.js:878]: Same with the element naming here.

Comment (shaargtz) [reply src/main/webapp/script.js]: done

File: src/main/webapp/script.js (hunk 1 above)
Comment (shaargtz) [reply src/main/webapp/script.js]: done

**Pull request merged**